templates/vite/.gitignore

```diff
@@ -29,3 +29,5 @@ coverage
 # Generated files
 public/mockServiceWorker.js
 .vitest-preview
+
+pnpm-lock.yaml
```

Comment:
cái pnpm lock này nên nằm ở ngoài, cụ thể là làm 1 cái .gitignore ở ngoài, e có thể bỏ toàn bộ các template vô 1 folder `templates` rồi ignore toàn bộ `templates/**/pnpm-lock.yaml`